src/jit/importer.cpp

```diff
@@ -5953,6 +5953,117 @@ GenTree* Compiler::impImportLdvirtftn(GenTree*                thisPtr,
     return gtNewHelperCallNode(CORINFO_HELP_VIRTUAL_FUNC_PTR, TYP_I_IMPL, helpArgs);
 }
 
+//------------------------------------------------------------------------
+// impBoxPatternMatch: match and import common box idioms
+//
+// Arguments:
+//   pResolvedToken - resolved token from the box operation
+//   codeAddr - position in IL stream adfter to mach
```

Comment:
typos? `after to match` (I'm not jit expert maybe has got some meaning if so sorry)

Reply:
Typo, no special meaning.